packages/database/src/runPostMigration.js

```diff
@@ -91,13 +91,6 @@ export const runPostMigration = async driver => {
     ),
   );
 
-  // Refresh analytics in case they've been impacted by migrations
-  console.log(`Migrations complete, refreshing analytics...`);
-  const start = Date.now();
-  await driver.runSql(`SELECT mv$refreshMaterializedView('analytics', 'public', true);`);
-  const end = Date.now();
-  console.log(`Analytics refresh took: ${end - start}ms`);
```

Comment:
Removed this as now we run the migrations via the `central-server` so the `AnalyticsRefresher` should be able to handle this 👍